custom-metrics-stackdriver-adapter/pkg/adapter/translator/query_builder.go

```diff
@@ -348,6 +414,7 @@ func quoteAll(list []string) []string {
 	return result
 }
 
+// Deprecated since FilterBuilder, use FilterBuilder instead
```

Comment:
Can we deprecate this function and replace all dependencies to use FilterBuilder?

Reply:
Yeah, eventually we will need to remove this function, but to give it a smooth deprecation process and avoid the risk of introduce breaking changes, I recommend mark it as deprecated and each time we only update those use cases we touch. 

Lastly, when all use cases of this function are replaced with `FilterBuilder`, then we remove this function (same for other deprecated function as well)

how do you think? 🤔